client/components/status/status.js

```
@@ -90,160 +90,184 @@ export default class Status extends React.Component {
+      this.notifyCpLogin(userData);
+      this.loginFormRef.current.submit();
+    } else if (!shouldLogin) {
+      if (captivePortalSyncAuth) {
+        this.handleLogin();
       }
+      this.finalOperations();
     }
+  };
+  async componentDidMount() {
+    this.initPage();
   }
```

ESLint

[error] Expected blank line between class members.

(lines-between-class-members)